pkg/controller/perconaservermongodb/pbm.go

```diff
@@ -209,10 +254,21 @@ func (r *ReconcilePerconaServerMongoDB) reconcilePiTRConfig(ctx context.Context,
 		return nil
 	}
 
-	stgName, _, err := cr.Spec.Backup.MainStorage()
-	if err != nil {
-		// no storage found
-		return nil
+	var stgName string
+	if cr.CompareVersion("1.20.0") >= 0 {
+		stgName, _, err = cr.Spec.Backup.MainStorage()
+		if err != nil {
+			// no storage found
+			return nil
+		}
+
+	} else {
+		if len(cr.Spec.Backup.Storages) == 1 {
+			for name := range cr.Spec.Backup.Storages {
+				stgName = name
+				break
+			}
+		}
 	}
```

Comment:
All this can be written to something simpler:


```
	var stgName string
	for name := range cr.Spec.Backup.Storages {
		stgName = name
		break
	}
	if cr.CompareVersion("1.20.0") >= 0 {
		stgName, _, err = cr.Spec.Backup.MainStorage()
		if err != nil {
			// no storage found
			return nil
		}
	}
```

I think it is the same behaviour, right?